test/dependency-manager/components.test.ts

```diff
@@ -319,7 +319,7 @@ describe('components spec v1', function () {
           }
         },
         dependencies: {
-          'concourse/ci': 6.2
+          'concourse/ci': '6.2'
```

Comment:
Does this break any of our examples?

Reply:
Nope, all of our examples use the `latest` tag